Dockerfile

```diff
@@ -1,4 +1,6 @@
-// Corresponding Dockerfile: https://github.com/JabRef/jabref/blob/main/Dockerfile.jabkit 
 FROM ghcr.io/jabref/jabkit:edge
 
-ENTRYPOINT ["/jabref/jabkit/bin/jabkit"]
+COPY entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
```

Comment:
No need for this. Use "git update-index chmod+x entrypoint.sh" and commit. That should also work on Windows then hopefully 😅